save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/info/UserPermissions.kt

```diff
@@ -0,0 +1,15 @@
+package com.saveourtool.save.info
+
+import kotlinx.serialization.Serializable
+
+/**
+ * @property isPermittedCreateContest permission for create contests
+ * @property isPermittedToBulkUpload permission for upload COSV files
+ * @property organizationName name of organization
+ */
+@Serializable
+data class UserPermissions(
```

Comment:
```suggestion
data class UserPermissionsInOrganization(
```

Reply:
I'm thinking of making this a more general class that won't just be tied to organizations.

Reply:
yes, but 
> user allowed to upload COSV and create contest only in organization